pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -119,10 +130,15 @@ func (b *ConsoleServerCLIConfigBuilder) clusterInfo() ClusterInfo {
 }
 
 func (b *ConsoleServerCLIConfigBuilder) authServer() Auth {
+	// we need this fallback due to the way our unit test are structured,
+	// where the ConsoleServerCLIConfigBuilder object is being instantiated empty
```

Comment:
not a bad thing I thing... It's good to have a fallback here...

Reply:
It can't function correctly without one or the other, correct?  Is it just because of unit test structure, or is the config broken without an entry?